src/patternfly/components/Pagination/examples/Pagination.md

```diff
@@ -131,13 +120,12 @@ import './Pagination.css'
 ```hbs
 {{#> pagination pagination--id="pagination-inset" pagination--modifier="pf-m-inset-none pf-m-inset-md-on-md pf-m-inset-2xl-on-lg"}}
   {{> pagination-total-items-content}}
-  {{> pagination-options-menu pagination-options-menu--id=(concat pagination--id '-options-menu')}}
+  {{> pagination-menu-toggle pagination-menu-toggle--id=(concat pagination--id '-menu-toggle')}}
   {{> pagination-nav-content pagination-nav--aria-label="Pagination nav - inset example"}}
 {{/pagination}}
 ```
 
 ## Documentation
-Note: `<button>` or `<a>` elements can be used in `.pf-v6-c-pagination__nav-page-select`.
```

Comment:
This wasn't accurate, as the element refers to the number input field for page select. We also don't allow customizing the elements for buttons used elsewhere in the component in React (menu toggle should be a button and the next/previous controls should be buttons)